Sources/MongoSwift/MongoClient.swift

```diff
@@ -167,8 +169,10 @@ public class MongoClient {
     /// Default maximum size for connection pools created by this client.
     internal static let defaultMaxConnectionPoolSize = 100
 
-    /// Indicates whether this client has been closed.
-    internal private(set) var isClosed = false
+    /// Indicates whether this client has been closed. We don't need a lock because:
```

Comment:
Is this wording intentional for public facing docs? could be confusing, the driver doesn't need a lock or the user doesn't need a lock.

Reply:
this is a private variable, so it's just for our own reference. that said I've opened it to say "a lock around this variable is not needed" so there's less confusion around who "we" is